index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>CV</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>Vladislav Lisetskii</h1>
+    <hr>
+    <div class="info">
+      <div class="personal-info">
+        <ul>
+          <li>Position: Fullstack developer (Java/Javascript)</li>
+          <li>Age: 25</li>
+          <li>City: Moscow</li>
+        </ul>
+      </div>
+      <div class="contacts">
+        <ul>
+          <li>E-mail: <a href="mailto:vladlis54@gmail.com">vladlis54@gmail.com</a></li>
+          <li>Phone: <a href="phoneto:88005553535">8 (800) 555-35-35</a></li>
+          <li>Github: <a href="https://github.com/Vladlis">Vladlis</a></li>
+        </ul>
+      </div>
+    </div>
+    <img src="assets/photo.jpeg" alt="photo" class="photo">
+    <hr>
+  </header>
+  <section class="summary">
+    <h2>Summary</h2>
+    <span>I am a Java developer who became tired of constant enterprise development hell, so I have mastered frontend
+      development to have more fun. I have lots of experience in processes and methodologies obtained during numerous
+      projects for big and small clients. I am seeking of a well-paid and interesting position as a fullstack
+      developer.</span>
+      <hr>
+  </section>
+  <section class="skills">
+    <h2>Skills</h2>
+    <ul>
+      <li>Highly expirienced in remote work and have good communication skills.</li>
+      <li>HStrong abilitity to manage legacy code and adapt for customer needs.</li>
+      <li>Key skills: Core Java, SQL, SVN, Git, JUnit, TDD, Maven, CI, Checkstyle, JBoss Fuse, Spring DI, Spring Cloud</li>
+      <li>Frontend: js, html, css</li>
+    </ul>
+    <hr>
+  </section>
+  <section class="projects">
+    <h2>Latest project</h2>
+    <h4>2018-now Java Tech Lead</h4>
+    <span>Big project with big responsibilities. I have roles of technical lead in a team of 3-5 people.</span>
+    <h2>One more project</h2>
+    <h4>2017-2018 Java Developer</h4>
+    <span>Foreign project with old technologies. I've go a lot of managing experience here.</span>
+    <h2>One of the first projects</h2>
+    <h4>2016-2017 Junior Java Developer</h4>
+    <span>A lot of boring stuff for juniors as always.</span>
+    <hr>
+  </section>
+  <section class="Education">
+    <h2>Sevastopol State University</h2>
+    <h4>2010-2016 Maters Degree</h4>
+    <span>Masters degree in computer engineering.</span>
+    <h2>English courses</h2>
+    <h4>English school</h4>
+    <span>Got B2 FCE certificate as a result.</span>
+    <hr>
+  </section>
+  <footer class="footer">
+    <span>Have a nice day :)</span>
+  </footer>
+</body>
+</html>
```

Comment:
not like I care too much about empty lines, anyway it's a general guideline to have them
I guess you may just configure your editor to insert them automatically